config.py

```diff
@@ -13,6 +14,7 @@ def envvar(key, default=None):
     return default
 
 def config(app):
+    load_dotenv()
```

Comment:
I don't want this plugin to assume a `.env` file exists in the plugin directory. CTFd operators will set environment variables in a number of ways, including through Docker or using the `export` command in `bash`. Having this always load the `.env` file could result in confusing behavior for operators.